packages/pds/src/account-manager/index.ts

```diff
@@ -71,6 +68,22 @@ export class AccountManager {
     return got?.did ?? null
   }
 
+  async getAccountStatus(handleOrDid: string): Promise<AccountStatus> {
+    const got = await this.getAccount(handleOrDid, {
+      includeDeactivated: true,
+      includeTakenDown: true,
+    })
+    if (!got) {
+      return AccountStatus.Deactivated
```

Comment:
Is this one intended to be `AccountStatus.Deleted`?

Reply:
ah yup good snag 👍